R/generate_dist.R

```diff
@@ -130,13 +130,17 @@
     )
   )
 
+  if (args$dist == "trtAssign") {
+    newColumn <- as.integer(newColumn)
+  }
+  
```

Comment:
I think it would be better to modify the `trtAssign` function [directly](https://github.com/kgoldfeld/simstudy/blob/403a6adadf6bb34bf396a68b4ab622a5c3fcd1b2/R/group_data.R#L354), rather than including it here. That way if users generate treatment assignments more generally (it can be done outside of `defData`) will also be constrained to integer results.

Reply:
Just updated this